aperturedb/Connector.py

```diff
@@ -496,16 +496,15 @@ def _renew_session(self):
                 time.sleep(1)
                 count += 1
 
-    def create_new_connection(self) -> Connector:
+    def clone(self) -> Connector:
```

Comment:
this is likely to break apps/demos/benchmarks. If this change is introduced, please make sure to make the changes on other repos.

Reply:
or we can do what we did with the other things, make this a wrapper that throws depricated, and then make a log parser which finds depricated warnings? I think this might be useful in the future too.

Reply:
added the deprecated func as a wrapper with warning.